lib/views/Table.js

```diff
@@ -53,6 +54,16 @@ Table.prototype.tabularise = function () {
     var data = this.data;
     var valueAttr = yAxis.key || this.moduleConfig['value-attribute'];
 
+    //sort the data if there is a sortby
+    if (sortBy) {
+      var sortOrder = this.moduleConfig['sort-order'] ?
+        this.moduleConfig['sort-order'] : 'descending';
```

Comment:
I did some data digging. Of the 65 tables:
59 descending
6 ascending

So I guess yes....